lib/finch/util.ex

```diff
@@ -0,0 +1,30 @@
+defmodule Finch.Util do
+  @moduledoc false
+
+  def maybe_log_secrets(:https, socket) do
+    maybe_log_secrets(:https, System.get_env("SSLKEYLOGFILE"), socket)
```

Comment:
I don't think we should read the system env on every single https request. Couldn't we add a new conn_opt instead?

Reply:
A configuration setting makes more sense!